fontscan/fontmap.go

```diff
@@ -334,6 +334,22 @@ func (fm *FontMap) FindSystemFont(family string) (Location, bool) {
 	return Location{}, false
 }
 
+// MatchedSystemFonts return all matched fonts
```

Comment:
What about 

// `FindSystemFonts` is the same as `FindSystemFont`, but returns all matched fonts.